<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QPushButton" name="btn_start">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stop">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QLabel" name="ans_label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>981</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="que_label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>150</y>
         <width>981</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QPushButton" name="btn_update">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>210</y>
         <width>461</width>
         <height>231</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>tcp端口</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>标点引擎</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>IP地址</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>快捷键</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>excel路径</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>默认值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>可用值</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="btn_rest">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_tip">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>431</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
